sdf.md

```diff
@@ -1015,9 +1013,10 @@ quality is absent.
 | sdfAction   | action    | zero or more named action definitions for this object                    |
 | sdfEvent    | event     | zero or more named event definitions for this object                     |
 | sdfData     | named-sdq | zero or more named data type definitions that might be used in the above |
+| sdfThing    | thing     | zero or more named thing definitions for this object                     |
```

Comment:
Hmm, I was wondering if it should it be possible to include `sdfThing`s in `sdfObject`s or if this does not make that much sense semantically.